app/models/ReturnLocation.scala

```diff
@@ -14,19 +14,28 @@
  * limitations under the License.
  */
 
-package controllers.auth
+package models
 
-import uk.gov.hmrc.play.frontend.auth.connectors.domain.Accounts
-import uk.gov.hmrc.play.frontend.auth.{AuthenticationProvider, TaxRegime}
 
-object AtedAgentRegime extends TaxRegime {
 
-  override def isAuthorised(accounts: Accounts): Boolean = {
-    accounts.agent.flatMap(_.agentBusinessUtr).isDefined
-  }
+import config.ApplicationConfig
+import play.api.libs.json.{JsString, Writes}
+import play.api.mvc.Call
+
+case class ReturnLocation(url: String, absoluteUrl: String)
```

Comment:
AMBER? - Unsure of the purpose of this file, I think it can be deleted.

Reply:
Done